package.json

```diff
@@ -42,17 +45,17 @@
     "vinyl-buffer": "^1.0.0",
     "vinyl-source-stream": "^1.1.0"
   },
-  "dependencies": {
-    "ampersand-app": "^1.0.4",
-    "ampersand-collection": "^1.4.5",
-    "ampersand-collection-view": "^1.4.0",
-    "ampersand-router": "^3.0.2",
-    "ampersand-state": "^4.5.6",
-    "ampersand-view": "^8.0.1",
-    "ampersand-view-switcher": "^2.0.0",
-    "bluebird": "^2.9.34",
-    "domready": "^1.0.8",
-    "mozilla-tabzilla": "^0.4.0",
-    "normalize.css": "^3.0.3"
+  "directories": {
+    "doc": "docs"
+  },
+  "homepage": "https://github.com/mozilla/idea-town#readme",
+  "license": "MPL-2.0",
+  "private": "true",
```

Comment:
Removed `main` attribute and added `private: true` since we aren't planning on deploying this to npm (as far as I know).